baricadr/model/repos.py

```diff
@@ -53,6 +53,21 @@ def __init__(self, local_path, conf):
 
                 self.freeze_age = conf['freeze_age']
 
+            self.auto_freeze = False
+            if 'auto_freeze' in conf and conf['auto_freeze'] is True:
+                self.auto_freeze = True
+                self.auto_freeze_interval = 7
+                if 'auto_freeze_interval' in conf:
+                    try:
+                        conf['auto_freeze_interval'] = int(conf['auto_freeze_interval'])
+                    except ValueError:
+                        raise ValueError("Malformed repository definition, auto_freeze_interval must be an integer in '%s'" % conf)
+
+                    if conf['auto_freeze_interval'] < 2 or conf['auto_freeze_interval'] > 10000:
+                        raise ValueError("Malformed repository definition, auto_freeze_interval must be an integer >1 and <10000 in '%s'" % conf)
```

Comment:
It's in seconds? it should be written here (and documented in the sample conf if it's not yet (too lazy to have a look now :D))

Reply:
It's in days. I can add a message (it's not specified either in freeze_age I believe)